### PR TITLE
Implement new speed measurement message

### PR DESCRIPTION
**Pull request by Kwuerch**

### CODE REVIEW

File: src/communication_pkg/scripts/HUD.py [hunk 1]
```diff
@@ -61,8 +61,10 @@ def speed_callback(data):
     Stores averaged values in info
     """
     global info, speed_bufl, speed_bufr, speed_idx
-    speed_bufl[speed_idx] = data.data[0]
-    speed_bufr[speed_idx] = data.data[1]
+
+    speed_bufl[speed_idx] = data.data
+    speed_bufr[speed_idx] = data.data
```
Comment (ssmarok) [inline src/communication_pkg/scripts/HUD.py:66]: May want to remove left/right distinction in future.

**Review by ssmarok — APPROVED** [review]
Looks good!

**Comment by ssmarok** [comment]
Feel free to merge when ready.

File: src/communication_pkg/scripts/HUD.py (hunk 1 above)
Comment (Kwuerch) [reply src/communication_pkg/scripts/HUD.py:66]: I moved the speed message to only have one value. This script is the only one that would have required a significant rewrite to utilize the new single valued message.

**Pull request merged**